src/engine/server/botnames.cpp

```diff
@@ -0,0 +1,127 @@
+#include "server.h"
+
+// C++11-style initializer
+std::vector<const char *> CServer::m_botnames = {
```

Comment:
Because I'll then need another variable m_botnamesSize, to hold the size of that static array.

sizeof(m_botnames) is known only inside botnames.cpp, but inside server.cpp sizeof(m_botnames) will cause a compilation error, and I don't want to move this big array into server.cpp.